src/lib/parquet/src/Flow/Parquet/ParquetFile/ColumnChunkReader/WholeChunkReader.php

```diff
@@ -105,7 +106,7 @@ private function readHeader($stream, int $pageOffset) : ?PageHeader
         try {
             \fseek($stream, $pageOffset);
             $thriftHeader = new \Flow\Parquet\Thrift\PageHeader();
-            $thriftHeader->read(new TCompactProtocol(new TBufferedTransport(new TPhpFileStream($stream))));
+            @$thriftHeader->read(new TCompactProtocol(new TBufferedTransport(new TPhpFileStream($stream))));
```

Comment:
What error you have silenced here?

Reply:
internal thrift warning about invalid array offset, unrelated for the logic, annoying in the CLI